dashboard/client/src/components/ActorTable.tsx

```diff
@@ -99,11 +91,36 @@ const ActorTable = ({
   const [actorIdFilterValue, setActorIdFilterValue] = useState(filterToActorId);
   const [pageSize, setPageSize] = useState(10);
 
+  const defaultSorterKey = "";
```

Comment:
let's extract this out of the component and name it DEFAULT_SORTER_KEY.

Let's also make the default state something like "actor-state" and make it one of the options in teh "Sort by" input so users can go back to that sort.

Reply:
> What does Sort by "All" mean in your screenshot?

"All" was a menu option added by default to the `SearchSelect` component to represent whatever the default sorting would be if no keys are specified by the user. This component is also used for the "State" filter on the clusters page where "All" would represent all states. I agree for this sort by input we should just specify one of the keys as the default for clarity.

I'm not sure having a separate sort by state is useful for the actors page given we have a "State" filter to either show alive or dead or all actors. If the state filter isn't set the current behavior of showing "Alive" actors at the top seems reasonable. The user specified sort by key would sort actors within the "Alive" and "Dead" state groups. 

Current behavior is to sort by reverse uptime as the default for the "All" option. I think sorting by "Uptime" (newest actors at top) probably makes more sense for many use cases so we can select that as the default key.